core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -239,11 +239,12 @@ private[spark] class Executor(
           val freedMemory = taskMemoryManager.cleanUpAllAllocatedMemory()
 
           if (freedMemory > 0) {
-            val errMsg = s"Managed memory leak detected; size = $freedMemory bytes, TID = $taskId"
+            val errMsg = s"Potential managed memory leak detected; " +
+              s"size = $freedMemory bytes, TID = $taskId.  See SPARK-14168."
```

Comment:
well, I was thinking that even its a WARN, a memory leak is still concerning to a user, so I wanted to explain to the user the cases where it was a known false-positive.  But I couldn't come up with a short yet adequate thing to throw into the msg, so I figured the full explanation was contained in the jira.

Sorry I should have mentioned that in the first place, and just changing to warn is fine in any case.